scripts/psijcli.py

```diff
@@ -23,12 +23,11 @@
                                         "flux",
                                         "lsf",
                                         "rp",
-                                        "saga",
```

Comment:
Not really a comment on this PR, but maybe executor names shouldn't be hardcoded.